docs/app/pages/components/components-sections/tables/column-sorting/column-sorting.component.html

```diff
@@ -110,6 +110,12 @@
         Defines if a column can be toggled to the <code>NoSort</code> state. If set to <code>false</code> this will
         toggle between the <code>Ascending</code> and <code>Descending</code> states when changing the sort direction.
     </tr>
+    <tr uxd-api-property name="ascendingIcon" type="string" defaultValue="ascend">
+        Specify an icon for sorting in ascending order.
```

Comment:
```suggestion
        Specify an icon name from the icon set for sorting in ascending order.
```

This should also contain a link to the icon set page, check the icon component documentation section to see how this is done.